addons/mail/static/src/core/common/message_reaction_list.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<template xml:space="preserve">
+    <t t-name="mail.messageReactionList">
+        <Dropdown state="preview" position="'bottom-start'" menuClass="'bg-view border-0 p-0 mt-1 overflow-visible rounded-3'" arrow="true" manual="true">
+            <t t-call="mail.reactionButton" />
+            <t t-set-slot="content">
+                <t t-call="mail.reactionMenuListPreview" />
+            </t>
+        </Dropdown>
+    </t>
+    
+    <t t-name="mail.reactionButton">
+        <button class="position-relative o-mail-MessageReaction btn d-flex p-0 border rounded-2 mb-1"
+            t-on-click="() => this.onClickReaction(props.reaction)"
+            t-on-contextmenu="onContextMenu"
+            t-att-class="{
+                'o-selfReacted border-primary text-primary fw-bolder': this.hasSelfReacted(props.reaction),
+                'bg-view': !this.hasSelfReacted(props.reaction),
+                'ms-1': env.inChatWindow and env.alignedRight,
+                'me-1': !(env.inChatWindow and env.alignedRight),
+            }"
+            t-ref="reactionButton">
+                <span class="mx-1" t-esc="props.reaction.content" />
+                <span class="mx-1" t-esc="props.reaction.count" />
+        </button>
+    </t>
+    
+    <t t-name="mail.reactionMenuListPreview">
+        <div
+            class="o-mail-messageReactionMenuList px-0 py-1 shadow border rounded-3 border-end-0"
```

Comment:
`shadow` was too strong, so I put `shadow-sm` in menuClass instead. Also removed `rounded-3` because too much roundness gives a childish look. It's ok on some elements like bubble, but usually prefer basic `rounded` or none at all